app/client/components/helpCentre/helpCentreArticle.tsx

```diff
@@ -53,16 +52,20 @@ const HelpCentreArticle = (props: HelpCentreArticleProps) => {
       );
   }, [props.articleCode]);
 
-  const selectedNavTopic = helpCentreNavConfig.find(
-    topic => topic.id === article?.topics[0].path
-  );
+  const setSelectedTopicObject = React.useContext(SelectedTopicObjectContext);
+  useEffect(() => {
+    const selectedNavTopic = helpCentreNavConfig.find(
+      topic => topic.id === article?.topics[0].path
+    );
```

Comment:
I don't think you need this bit anymore because you pass `selectedNavTopic.id` on line 61. You're ok passing `article?.topics[0].path` directly (topic ID and path are synonyms on this system). Not having this check would remove the need to import `helpCentreNavConfig` on this file.